### PR TITLE
Release 0.8.0

### PR DESCRIPTION
**Pull request by UncleRus**
@trombik What do you think of the CHANGELOG format?
And I might be missing something, could you please check?

### CODE REVIEW

**Comment by UncleRus** [comment]
BTW, new CI is awesome!

**Comment by trombik** [comment]
there are several solutions to reduce burden to release.

https://github.com/marvinpinto/action-automatic-releases is a fully automatic action, while https://github.com/release-drafter/release-drafter is semi-auto.

**Comment by UncleRus** [comment]
> https://github.com/marvinpinto/action-automatic-releases is a fully automatic action, while https://github.com/release-drafter/release-drafter is semi-auto.

I'll take a look, thank you.

But what about this release? Do you approve?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+- (ultrasonic) Added more precise measurement functions
+- (led_strip_spi) Fixed SPI buffer limitation for ESP8266
+- (led_strip) Fixed buffer overflow error for RGBW strips
+- (led_strip) White component for RGBW strips now calculated automatically
+- (led_strip) Fixed bug in led_strip_set_pixels()
+- (led_strip) Added support for global brightness for ESP-IDF >= v4.4
+- (refactoring) Updated component makefiles and CMakeLists
+- (i2cdev) Added option to disable all mutexes
+
+
```
Comment (trombik) [inline CHANGELOG.md:31]: double empty lines should be avoided. I will import `markdownlint` workflow for `*.md` later to check markdown files in the CI process.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## v.0.8.0
+
+#### Features
```
Comment (trombik) [inline CHANGELOG.md:5]: a ${N} heading should be a child of ${N - 1} (should be `###` in this case).

**Review by trombik — COMMENTED** [review]
nothing other than minor comments I left.

in general, when reading a release note, I usually do so with some questions like:

- why should I upgrade?
- does it break old code when I upgrade? what should I do when it does?

featuring new code ("now we have `foo` driver!), and notes about backward compatibility might be nice to have.

**Pull request merged**